Auth_ERROR_README.md

```diff
@@ -0,0 +1,47 @@
+## Difficulty Logging In on Local Machine
+
+### Issue
+When attempting to authenticate using a Fedora login in a Flask application, an AttributeError is raised due to the usage of the deprecated `encodestring` attribute of the `base64` module. Additionally, after the app runs, the user faces server errors when logging in due to HTTPS configuration.
+
+### Description
+The AttributeError occurs because the `encodestring` attribute has been deprecated in Python 3 and replaced with `encodebytes`. Attempting to use `encodestring` results in an AttributeError. Furthermore, server errors occur when logging in due to the Flask application being configured to run over HTTPS.
+
+### Cause
+The root cause of the AttributeError is the usage of the deprecated `encodestring` attribute instead of `encodebytes` in the affected file. The server errors occur because the Flask application is configured to run over HTTPS by default.
+
+### Solution
+To resolve the AttributeError, replace `encodestring` with `encodebytes` in the affected file (`storage.py` within the `social_sqlalchemy` package). To resolve the server errors when logging in, change the Flask application configuration to run over HTTP instead of HTTPS.
+
+### Steps to Resolve
```

Comment:
This looks like the issue is with old `social_sqlalchemy` dependency. I will look into that.

Reply:
I would recommend to remove this file altogether and I need to fix the dev environment as I wasn't aware that there are issues.

Reply:
Okay, I just removed them